modules/db/src/blaze/db/api.clj

```diff
@@ -408,14 +401,9 @@
 
 (defn total-num-of-type-changes
   "Returns the total number of changes (versions) of resources with the given
-  `type` starting as-of `db`.
-
-  Optionally a `since` instant can be given to define a point in the past where
-  the calculation should start."
-  ([db type]
-   (p/-total-num-of-type-changes db type nil))
-  ([db type since]
-   (p/-total-num-of-type-changes db type since)))
+  `type` starting as-of `db`."
+  [db type]
+  (p/-total-num-of-type-changes db type))
```

Comment:
as a cleanup, please create the `tid` here using `(codec/tid type)` instead of passing the `type`. we have always only the `tid` at the DB level